wallet/wallet.go

```diff
@@ -3668,11 +3672,12 @@ func decodeStakePoolColdExtKey(encStr string,
 
 // Open loads an already-created wallet from the passed database and namespaces.
 func Open(db walletdb.DB, pubPass []byte, cbs *waddrmgr.OpenCallbacks,
-	voteBits uint16, stakeMiningEnabled bool, balanceToMaintain float64,
-	addressReuse bool, rollbackTest bool, pruneTickets bool, ticketAddress string,
-	ticketMaxPrice float64, ticketBuyFreq int, poolAddress string,
-	poolFees float64, ticketFee float64, addrIdxScanLen int, stakePoolColdExtKey string,
-	autoRepair, allowHighFees bool, relayFee float64, params *chaincfg.Params) (*Wallet, error) {
+	voteBits uint16, voteBitsExtended string, stakeMiningEnabled bool,
```

Comment:
The reason for this is because historically this function has received the raw output from the initial configuration file and then done the parsing of that data in this function. The re-parsing of the parsed configuration file could be moved to another function in a refactor, but that's outside the scope of this PR.